core/src/stack/outputs.rs

```diff
@@ -1,3 +1,7 @@
+use miden_crypto::Word;
+
```

Comment:
typically we try and avoid spacing between imports.

Reply:
This can be solved by the [`group_imports`](https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#group_imports) section of rustfmt.toml. I see that it's been commented out, and [the reason](https://github.com/0xPolygonMiden/miden-vm/pull/594) seems to be that the CI complained that the features were on nightly only. This is solved by configuring the CI to use nightly rust for the `cargo fmt` command. I will open another PR for it

Reply:
#1158